app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -31,6 +31,7 @@ Fae.form.filtering = {
     // hardcode _this because this === Fryr object
     var _this = Fae.form.filtering;
     var post_url = _this.$filter_form.attr('action');
+    $('.js-sort-row').css({opacity: 0.5});
```

Comment:
@jasonfine I'd rather not apply CSS with JS. Can you apply a class with JS and use CSS on the class?

thanks!